static/simple-portfolio-template/index.html

```diff
@@ -0,0 +1,344 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;600;800&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>A Simple Portfolio Template</title>
+    <script type="module" defer src="./js/main.js"></script>
+  </head>
+
+  <body>
+    <section
+      class="bg-hero-bg relative bg-cover bg-fixed bg-center text-white"
+      id="hero-section"
+    >
+      <article class="navbar">
```

Comment:
Maybe a header would be more semantically suitable here?

Reply:
Perfect. I will separate the header and hero section and put them in relevant tags, for better structure.